features/step_definitions/files_steps.rb

```diff
@@ -26,7 +41,6 @@
 end
 
 
-Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |file_name, content|
-  expect(uncommitted_files).to eql [file_name]
-  expect(IO.read file_name).to eql content
+Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |name, content|
```

Comment:
What about `Then my workspace (?:still|again) has an uncommitted file with name: "([^"]+)" and content: "([^"]+)"`? Sorry about the shotgun refactoring this requires. If its too much work, we can also file a separate ticket for this, and I'll do it very efficiently using Vim.